generator/src/test/java/com/scottlogic/deg/generator/builders/AtomicConstraintBuilder.java

```diff
@@ -21,60 +21,56 @@
 import com.scottlogic.deg.common.profile.constraints.atomic.IsInSetConstraint;
```

Comment:
Maybe rename this file `TestAtomicConstraintBuilder` if you are calling a class it uses extensively `Test...`? That's assuming that you are not using this in any main/non-test classes.